index.html

```diff
@@ -206,5 +206,83 @@ <h4>Skills</h4>
         
     </section>
 </section>
+
+<footer>
+    <div class="images">
+        <div class="space">
+           <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
+           <div class="image02"><img src="assets/Shape_1.png" alt="#"  class="shape01"></div>
+           </div>
```

Comment:
- [ ] Let's make sure we use these shapes as a background image using `CSS` as long as they are used for styling purposes